src/check_expr.cpp

```diff
@@ -1409,11 +1410,16 @@ gb_internal bool is_polymorphic_type_assignable(CheckerContext *c, Type *poly, T
 			    poly->Struct.soa_kind != StructSoa_None) {
 				bool ok = is_polymorphic_type_assignable(c, poly->Struct.soa_elem, source->Struct.soa_elem, true, modify_type);
 				if (ok) switch (source->Struct.soa_kind) {
-				case StructSoa_Fixed:
 				default:
 					GB_PANIC("Unhandled SOA Kind");
 					break;
-
+				case StructSoa_Fixed:
+					if (modify_type) {
+						bool breakpoint = true;
```

Comment:
Please remove debug code.

Reply:
Now that I look at it, there is an unhandled case remaining. I think might actually better if I keep it there and explicitly put in the unhandled case of `StructSoa_None` instead (See types.cpp:125). Even though the rest of the cases are still now handled, it might be better to have that variant to be explicit in this switch. If it is an invalid case, crashing here would retain the existing behavior for that case, even if it may never be hit.